nequip/nn/_atomwise.py

```diff
@@ -171,6 +171,12 @@ def forward(self, data: AtomicDataDict.Type) -> AtomicDataDict.Type:
         return data
 
     def update_for_rescale(self, rescale_module):
+        if hasattr(rescale_module, "related_keys"):
+            if not (
+                self.field in rescale_module.related_keys
+                or self.out_field in rescale_module.related_keys
+            ):
+                return
```

Comment:
small (rare) issue: if the key is globally shifted but not scaled, this will still rescale it?

Reply:
it depends on what the call-back function is doing there right. if there's no global scale, there is also not value to divide?

Reply:
right... though it is _possible_ for there to be a scale and a shift, but for a key to be only in the shift and not the scale